lib/record_store/provider/ns1.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'ns1/client'
+require 'net/http'
```

Comment:
Reminder: Move this where the refinement / monkey patch goes.

Reply:
Right, this doesn't need to be here. Thanks!